instrumentation/aws-sdk/aws-sdk-1.11/src/main/java/io/opentelemetry/auto/instrumentation/awssdk/v1_11/AWSHttpClientInstrumentation.java

```diff
@@ -74,14 +74,12 @@ public static void methodExit(
         @Advice.Argument(value = 0, optional = true) final Request<?> request,
         @Advice.Thrown final Throwable throwable) {
       if (throwable != null) {
-        SpanWithScope spanWithScope = request.getHandlerContext(SPAN_SCOPE_PAIR_CONTEXT_KEY);
-        if (spanWithScope != null) {
+        SpanWithScope scope = request.getHandlerContext(SPAN_SCOPE_PAIR_CONTEXT_KEY);
+        if (scope != null) {
           request.addHandlerContext(SPAN_SCOPE_PAIR_CONTEXT_KEY, null);
-          Span span = spanWithScope.getSpan();
-          DECORATE.onError(span, throwable);
-          DECORATE.beforeFinish(span);
-          span.end();
-          spanWithScope.closeScope();
+          Span span = scope.getSpan();
+          ERROR_TRACER.endExceptionally(span, throwable);
```

Comment:
I think reads a little nicer
```suggestion
          ERROR_TRACER.endExceptionally(scope.getSpan(), throwable);
```

Reply:
Fixed in #922.  There is no longer ERROR_TRACER.